src/reference_implementations/prompting_vector_llms/README.MD

```diff
@@ -88,4 +88,4 @@ One of the most effective ways to help LLMs perform such tasks is through reason
 
 ## Can we transfer gradient optimized prompts to Falcon?
 
-In the folder `transferring_gradient_optimized_prompts/`, we consider whether the gradient optimized prompts that we have found during discrete prompt optimization through gradient based search for the T5 model are transferrable to the Falcon 7B model for sentiment analysis. This model is larger than T5, but has a different tokenizer and a different architecture. T5 has an encoder-decoder coupling, while Falcon is decoder-only. The prompts were optimized using AutoPrompt with a Vector implementation [here](https://github.com/VectorInstitute/PromptEngineering/blob/main/src/reference_implementations/prompt_zoo/gradient_search.py). Due to the compute intensive nature and complexity of the code, the implementation source code has not been included in the lab.
+In the folder `transferring_gradient_optimized_prompts/`, we consider whether the gradient optimized prompts that we have found during discrete prompt optimization through gradient based search for the T5 model are transferrable to the LLaMA-2 7B model for sentiment analysis. This model is larger than T5, but has a different tokenizer and a different architecture. T5 has an encoder-decoder coupling, while Falcon is decoder-only. The prompts were optimized using AutoPrompt with a Vector implementation [here](https://github.com/VectorInstitute/PromptEngineering/blob/main/src/reference_implementations/prompt_zoo/gradient_search.py). Due to the compute intensive nature and complexity of the code, the implementation source code has not been included in the lab.
```

Comment:
You mean "LLaM2-2 is decoder-only" instead of Falcon perhahsp?

Reply:
Yes! I was originally going to use falcon, but forgot I needed activations.